### DISCUSSION TITLE
Searching a show's recordings - options, how to?

### DISCUSSION THREAD

**Discussion by Sagetvmaster** [OpenSageTV/Community#64] [General]
I'm collecting the entire 'Mayday' series.  There are 22 seasons and over 200 episodes.
Is there a way in Sage to display the recordings by sequential seasons/episodes, as opposed to alphabetically?

**Comment by LoisGNS** [comment]
That's how I have things set by default. I use the Malore display in the stock STV.  Group by title, sort by Orig air date (which coincides with seasons/episodes).
